src/components/code/index.tsx

```diff
@@ -46,7 +48,11 @@ const Code: FC<CodeProps> = ({
   const theme = useTheme<ITheme>();
 
   const useCodeKey = () => {
-    setCopied(copyToClipboard(content));
+    if (copyCallback) {
+      setCopied(copyCallback());
```

Comment:
doesn't in make sense to pass `content` here too since maybe the content for callback could be useful?

Reply:
But the content is sent with the copyCallback method, so I do not see the use